ocfweb/docs/docs/staff/procedures/accounts/renaming.md

```diff
@@ -0,0 +1,151 @@
+[[!meta title="Renaming Accounts"]]
+
+Sometimes it's desirable to rename an OCF account.
```

Comment:
Now that it's been said multiple times, I think the first paragraph should state that, in most cases, the correct thing to do would be sorry the account and create a new one. Renaming an account is really only useful if there was an error during account creation; I would never actually attempt to rename an account that actually has cron jobs, a database, numerous symlinks, etc.